h2/src/main/org/h2/util/DateTimeFunctions.java

```diff
@@ -374,6 +381,161 @@ public static Value extract(String part, Value value) {
         return result;
     }
 
+    /**
+     * Truncate the given date to 'day'
+     *
+     * @param timeUnit the time unit (e.g. 'DAY', 'HOUR', etc.)
+     * @param value the date
+     * @return date truncated to 'day'
```

Comment:
comment is wrong now should be something like "truncate the given date to the unit specified"